doc/md/versioned/execution.mdx

```diff
@@ -0,0 +1,156 @@
+---
+id: execution
+slug: /versioned/execution
+title: Migration Execution
```

Comment:
Maybe `Applying Migration`? Because that's the name of the command and this is how we name it here: https://atlasgo.io/declarative/apply